src/test/java/org/lobid/resources/Hbz01MabXml2ElasticsearchLobidTest.java

```diff
@@ -272,16 +273,23 @@ static private SearchResponse getElasticsearchDocuments() {
 		static String getAsNtriples() {
 			return Arrays.asList(getElasticsearchDocuments().getHits().getHits())
 					.parallelStream()
-					.flatMap(hit -> Stream.of(toRdf(hit.getSourceAsString())))
+					.flatMap(
+							hit -> Stream.of(toRdf(cleanseEndtime(hit.getSourceAsString()))))
 					.collect(Collectors.joining());
 		}
 
 		private static void getAsJson() {
 			for (SearchHit hit : getElasticsearchDocuments().getHits().getHits()) {
-				stripContextAndSaveAsFile(hit.getSourceAsString());
+				stripContextAndSaveAsFile(cleanseEndtime(hit.getSourceAsString()));
 			}
 		}
 
+		private static String cleanseEndtime(String jsonld) {
+			return jsonld.replaceFirst(
+					"\"endTime\":\"\\d\\d\\d\\d-\\d\\d-\\d\\dT\\d\\d:\\d\\d:\\d\\d\"",
```

Comment:
We could shorten this a bit with quantifiers: `"\"endTime\":\"\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\"",`